recipes/zulu-openjdk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,15 @@
+from conans import ConanFile
+import subprocess
+
+class TestPackage(ConanFile):
+
+    def build(self):
+        pass # nothing to build, but tests should not warn
+
+    def test(self):
+        output = subprocess.run(['java', '--version'], stdout=subprocess.PIPE)
```

Comment:
why not using [`self.run` `with output` argument](https://docs.conan.io/en/latest/reference/conanfile/other.html#running-commands)  ?

Reply:
it's a while since I wrote this, but very likely because I reused code from a other python script that has no access to conan tools.
Do you think using conan tools will improve the test, and that it's worth to pull in now ?

Reply:
It won't change anything to the test, but most readers seeing `subprocess.run` will ask themselves why didn't you use the facility provided by conan (if there is a good reason, it should be explained in a comment)

Reply:
ok, changed as it is in the docs